mobile/android/base/android_strings.dtd

```diff
@@ -168,15 +174,20 @@ selected to be added to the home page. -->
 <!ENTITY pref_cookies_disabled "Nie akceptuj">
 
 <!ENTITY pref_tracking_protection_title "Ochrona przed śledzeniem">
-<!ENTITY pref_tracking_protection_summary "Blokowanie przez program &brandShortName; prób śledzenia aktywności użytkownika przez strony">
+<!ENTITY pref_tracking_protection_summary3 "Włączona w trybie prywatnym">
 <!ENTITY pref_donottrack_title "Ograniczanie śledzenia">
 <!ENTITY pref_donottrack_summary "Informuj witryny, że użytkownik nie chce być śledzony">
 
-<!ENTITY tab_queue_toast_message2 "Karta skolejkowana w programie &brandShortName;">
+<!ENTITY tracking_protection_prompt_title "Nowa ochrona przed śledzeniem">
+<!ENTITY tracking_protection_prompt_text "Aktywne blokowanie elementów śledzących aktywność użytkownika.">
+<!ENTITY tracking_protection_prompt_tip_text "Więcej informacji w ustawieniach prywatności">
+<!ENTITY tracking_protection_prompt_action_button "Zamknij">
+
+<!ENTITY tab_queue_toast_message3 "Karta zapisana w programie &brandShortName;">
```

Comment:
I'm not sure if "zapisana" is best choice here, maybe "zachowana" or even "dodana do programu" would better fit?